Build.PL

```diff
@@ -289,6 +289,7 @@ my $requires = {
                 'Digest::SHA1'                    => '0.0',
                 'English'                         => '1.02',
                 'MIME::Base64'                    => 0,
+                'perl'                            => '5.14',
```

Comment:
perl Build.PL 
Checking prerequisites...
  requires:
    !  perl (5.16.3) is installed, but we need version >= 5.14

'perl' => '5.14.0' is OK